loki/transform/transform_remove_code.py

```diff
@@ -0,0 +1,333 @@
+# (C) Copyright 2018- ECMWF.
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+# In applying this licence, ECMWF does not waive the privileges and immunities
+# granted to it by virtue of its status as an intergovernmental organisation
+# nor does it submit to any jurisdiction.
+
+"""
+Collection of utilities to automatically remove code elements or
+section and to perform Dead Code Elimination.
+"""
+
+from loki.expression.symbolic import simplify
+from loki.tools import flatten, as_tuple
+from loki.ir import Conditional, Transformer, Comment
+from loki.pragma_utils import is_loki_pragma, pragma_regions_attached
+from loki.transform.transformation import Transformation
+
+
+__all__ = [
+    'RemoveCodeTransformation',
+    'do_remove_dead_code', 'RemoveDeadCodeTransformer',
+    'do_remove_marked_regions', 'RemoveRegionTransformer',
+    'do_remove_calls', 'RemoveCallsTransformer'
+]
+
+
+class RemoveCodeTransformation(Transformation):
+    """
+    A :any:`Transformation` that provides named call and import
+    removal, code removal of pragma-marked regions and Dead Code
+    Elimination for batch processing vis the :any:`Scheduler`.
+
+    The transformation will apply the following methods in order:
+    * :method:`do_remove_calls`
+    * :method:`do_remove_marked_regions`
+    * :method:`do_remove_dead_code`
```

Comment:
The markup is not producing the right appearance: https://sites.ecmwf.int/docs/loki/276/loki.transform.transform_remove_code.html#module-loki.transform.transform_remove_code

I think the keyword to create the link is `:meth:`, or simply `:any:` should work as well.
Not sure why the list isn't rendered as such, maybe requires an empty line before the first item?